extensions/panache/hibernate-orm-rest-data-panache/deployment/src/test/java/io/quarkus/hibernate/orm/rest/data/panache/deployment/entity/PanacheEntityResourceGetMethodTest.java

```diff
@@ -29,4 +29,20 @@ void shouldCopyAdditionalMethodsAsResources() {
                 .and().body("name", is("full collection"));
     }
 
+    @Test
+    void shouldReturnItemsForFullCollection() {
+        given().accept("application/json")
+                .when().get("/items?collection=full")
```

Comment:
collection is an entity class, so I don't understand how `collection=full` can be resolved, I think it should be `collection.name=full` or at least it would make more sense for users.